installation/scripts/install-db.sh

```diff
@@ -50,6 +50,9 @@ do
 done
 set -- "${POSITIONAL[@]}" # restore positional parameters
 
+echo "Wait for helm stable status"
+wait_for_helm_stable_state "localdb" "compass-system" 
```

Comment:
When job is started it does not know how many commits are coming and when. It tries its best to provision the VM and complete the installation. In case Prow get notification for new commit - it will send kill signal to this job and if the initial provisioning was not completed the VM will be cleared. In case this kill signal comes to VM during compass installation or tests (after finishing base installation) the VM will be reused for the next commit and cleanup process will only uninstall compass instead clearing the whole VM.